core/utils/silly-i18n/src/exceptions.js

```diff
@@ -0,0 +1,52 @@
+/**
```

Comment:
Nit: In JavaScript it's AFAIK generally not called _exception_, but _error_ (in regards to the file name).